website/docs/wallet/withdraw-validator.md

```diff
@@ -0,0 +1,404 @@
+6. Next you will be asked for your **withdrawal credentials,** which you should now have if you followed this guide
+7. Next you will be asked for the Ethereum address you wish to use to receive your withdrawn funds. This needs to be checksummed, and you can get it from your wallet or a block explorer. **You cannot change this once it is set on-chain**, so triple check it
+
+Below is an example of running through the interactive process explained above:
+
+```
+python ./staking_deposit/deposit.py generate-bls-to-execution-change
+Please choose your language ['1. العربية', '2. ελληνικά', '3. English', '4. Français', '5. Bahasa melayu', '6. Italiano', '7. 日本語', '8. 한국어', '9. Português do Brasil', '10. român', '11. Türkçe', '12. 简体中文']:  [English]: english
+
+Please choose the (mainnet or testnet) network/chain name ['mainnet', 'goerli', 'sepolia', 'zhejiang']:  [mainnet]: zhejiang
+
+Please enter your mnemonic separated by spaces (" "). Note: you only need to enter the first 4 letters of each word if you'd prefer.: 
+bike shoe attitude violin fun life punch enhance attend bright voyage wheel clutch taxi high health siren jealous tell female upon firm manual wage
+
+Please enter the index (key number) of the signing key you want to use with this mnemonic. [0]: 0
+
+Please enter a list of the validator indices of your validator(s). Split multiple items with whitespaces or commas.: 8
+
+Please enter a list of the old BLS withdrawal credentials of your validator(s). Split multiple items with whitespaces or commas.: 00a6bd30000296e9c9f5823b09e689ff0bc0b1bea1d256caab9a5f213a226b33
+
+Please enter the 20-byte execution address for the new withdrawal credentials. Note that you CANNOT change it once you have set it on chain.: 0x9B984D5a03980D8dc0a24506c968465424c81DbE
+
+**[Warning] you are setting an Eth1 address as your withdrawal address. Please ensure that you have control over this address.**
+
+Repeat your execution address for confirmation.: 0x9B984D5a03980D8dc0a24506c968465424c81DbE
+
+**[Warning] you are setting an Eth1 address as your withdrawal address. Please ensure that you have control over this address.**
+
+Success!
+Your SignedBLSToExecutionChange JSON file can be found at: /home/me/Desktop/code/python/staking-deposit-cli/bls_to_execution_changes
+```
+
+### Step 3: Verify your output
+
+Once you complete the above, you’ll have a file contained in the `bls_to_execution_changes/` folder of your [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli). It will represent a list of BLS to execution messages that have been signed with your private keys and are ready to submit to Ethereum. Here’s what a sample file of these looks like:
```

Comment:
Is this the actual output or was this run through `jq`?

Reply:
the actual output json file